### PR TITLE
Completed assignment 1 and updated labs 1 and 2

### PR DESCRIPTION
**Pull request by vhlopezch**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Added the completed notebook for Assignment 1 and updated lab files for training and evaluating CNN models.

## What did you learn from the changes you have made?
I learned how to build and improve a deep learning model using TensorFlow/Keras. I also understood how to measure accuracy and use metrics like Top-5 accuracy.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
Yes, I considered adding data augmentation or using a pretrained model, but decided to focus on batch normalization and dropout for now.

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Yes, I had issues running the notebook locally due to limited computing resources, so I used Google Colab to run and test the model.

## How were these changes tested?
The models were trained and evaluated in Google Colab using both training and test datasets provided.

## A reference to a related issue in your repository (if applicable)
N/A

## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

**Comment by vhlopezch** [comment]
Hi, I realized that I mistakenly worked on the wrong assignment notebook because I had forked the repository before it was updated with the correct materials.

Please disregard this pull request and refer to the updated and correct version here: [Assignment 1 v2 Pull Request](https://github.com/vhlopezch/deep_learning/pull/2)

Apologies for the confusion, and thank you for your understanding.
